.gitignore

```diff
@@ -31,6 +31,7 @@ platform_tools/android/apps/*/build
 platform_tools/android/apps/*/src/main/libs
 platform_tools/android/apps/*.hprof
 platform_tools/android/apps/*/.externalNativeBuild
+Makefile
```

Comment:
Can this be removed and can we append an RTC to the end of this to also pick up windows and qt build files too? For example, https://github.com/Esri/gdal/commit/6b04862ca35c657079620150f2206bee0ba9e464